apps/web/components/user-feedback/feedback-table.tsx

```diff
@@ -38,6 +39,10 @@ const FeedbackTable = ({ currentPage }: Props) => {
 
   if (error instanceof FetchError) clientAuthGuard(error.status);
 
+  const { tableHeads, tableRows } = useFeedbackTable({
+    data: data?.feedbacks || [],
```

Comment:
No fallback here `|| []` please, if no data, the new reference will be passed to `useFeedbackTable`

Handle the null/undefined values inside the hook